meilisearch-http/src/routes/indexes/documents.rs

```diff
@@ -181,9 +181,24 @@ async fn document_addition(
         Some("application/x-ndjson") => DocumentAdditionFormat::Ndjson,
         Some("text/csv") => DocumentAdditionFormat::Csv,
         Some(other) => {
-            return Err(MeilisearchHttpError::InvalidContentType(other.to_string()).into())
+            return Err(MeilisearchHttpError::InvalidContentType(
+                other.to_string(),
+                vec![
+                    "application/json".to_string(),
+                    "application/x-ndjson".to_string(),
+                    "application/csv".to_string(),
+                ],
+            )
+            .into())
+        }
+        None => {
+            return Err(MeilisearchHttpError::MissingContentType(vec![
+                "application/json".to_string(),
+                "application/x-ndjson".to_string(),
+                "application/csv".to_string(),
+            ])
+            .into())
```

Comment:
make it a const instead:
```const aACCEPTED_CONTENT_TYPE = [
                "application/json".to_string(),
                "application/x-ndjson".to_string(),
                "application/csv".to_string(),
            ];

// ... later
 return Err(MeilisearchHttpError::MissingContentType(ACCEPTED_CONTENT_TYPE.clone()))

Reply:
Here is the fix: https://github.com/meilisearch/MeiliSearch/pull/1769/commits/2f0768dcb03648a40aa32a5613041cd1caa80e32